src/transformState.js

```diff
@@ -4,8 +4,37 @@
  * @param {Object} state
  * @param {Object[]} actions
  */
+
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    const current = actions[i];
+
+    switch (true) {
```

Comment:
add `default` to switch to throw an error if there is no action type like that

Reply:
done